packages/react-core/src/components/Switch/Switch.tsx

```diff
@@ -48,15 +48,18 @@ class Switch extends React.Component<SwitchProps & OUIAProps, { ouiaStateId: str
     isChecked: true,
     isDisabled: false,
     isReversed: false,
-    'aria-label': '',
+    'aria-label': undefined,
+    'aria-labelledby': undefined,
     onChange: () => undefined as any
   };
 
   constructor(props: SwitchProps & OUIAProps) {
     super(props);
-    if (!props.label && !props['aria-label']) {
+    if (!props.label && !props['aria-label'] && !props['aria-labelledby']) {
       // eslint-disable-next-line no-console
-      console.error('Switch: Switch requires either a label or an aria-label to be specified');
+      console.error(
```

Comment:
I would as unit test for this.  That can be follow up.  If we want to get this in.